developer_manual/client_apis/WebDAV/basic.rst

```diff
@@ -78,28 +100,222 @@ You can request additional properties by sending a request body with the :code:`
-- :code:`{http://nextcloud.org/ns}has-preview`
-- :code:`{http://owncloud.org/ns}size` Unlike :code:`getcontentlength`, this property also works for folders reporting the size of everything in the folder.
-- :code:`{http://nextcloud.org/ns}rich-workspace` this property is provided by the text app
-- :code:`{http://nextcloud.org/ns}contained-folder-count` The number of folders directly contained in the folder (not recursively)
-- :code:`{http://nextcloud.org/ns}contained-file-count` The number of files directly contained in the folder (not recursively)
+A note about namespaces URI
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When building the body of your DAV request, you will request properties that are available under specific namespace URI.
+It is usual to declare prefixes for those namespace in the ``d:propfind`` element of the body.
+
+Here is the list of available namespace:
+
+=========================================  ======
+                   URI                     Prefix
+=========================================  ======
+DAV:                                       d
+http://owncloud.org/ns                     oc
+http://nextcloud.org/ns                    nc
+http://open-collaboration-services.org/ns  ocs
+http://open-cloud-mesh.org/ns              ocm
+=========================================  ======
+
+And here is how it should look in your DAV request:
+
+.. code-block:: xml
+
+	<?xml version="1.0"?>
+		<d:propfind
+			xmlns:d="DAV:"
+			xmlns:oc="http://owncloud.org/ns"
+			xmlns:nc="http://nextcloud.org/ns"
+			xmlns:ocs="http://open-collaboration-services.org/ns">
+			xmlns:ocm="http://open-cloud-mesh.org/ns">
+		...
+
+Supported properties
+^^^^^^^^^^^^^^^^^^^^
+
++-------------------------------+-------------------------------------------------+--------------------------------------------------------------------------+
+|           Property            |                   Description                   |                                 Example                                  |
++===============================+=================================================+==========================================================================+
+| <d:creationdate />            | The creation date of the node.                  | ``"1970-01-01T00:00:00+00:00"``                                          |
++-------------------------------+-------------------------------------------------+--------------------------------------------------------------------------+
+| <d:getlastmodified />         | The latest modification time.                   | ``"Wed, 20 Jul 2022 05:12:23 GMT"``                                      |
++-------------------------------+-------------------------------------------------+--------------------------------------------------------------------------+
+| <d:getetag />                 | The file's etag.                                | ``"&quot;6436d084d4805&quot;"``                                          |
```

Comment:
```suggestion
| <d:getetag />                 | The file's etag.                                | ``"6436d084d4805"``                                          |
```

Reply:
On purpose, the etag is wrapped in quotes

Reply:
Can you tell me why the &qout; is needed?

Reply:
I have harmonized the use of `"`. It was to denote that it is a string, but not really necessary.